packages/paste-core/components/typography/src/Paragraph.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+import {ParagraphProps} from './types';
+
+const handlePropValidation = ({children}: ParagraphProps): void => {
+  if (children == null) {
+    throw new Error(`[Paste: Typography Paragraph] Must have non-null children.`);
+  }
+};
+
+const Paragraph: React.FC<ParagraphProps> = props => {
```

Comment:
The name of the component here worthy of a poll of some sort? 

`Paragraph` might seem verbose to some, `P` is related to the HTML tag but might be not as clear.

Reply:
Polling the team. I found a few other design systems/ui-component libraries using `Paragraph`, which is part of why I went that route. I also thought using `P` could be confused with the standard `p`.

Reply:
`Paragraph` it is then 👍